Improve worker queues overview, other minor improvements (#1411)
astronomer/docs: astro/configure-worker-queues.md

## Worker queue settings

> Each worker queue on Astro can be configured with the following settings:

Reviewer: ```suggestion
You can configure each worker queue on Astro with the following settings:
```

These calculations are computed by KEDA every 10 seconds. For more information on how workers are affected by changes to a Deployment, read [What happens during a code deploy](deploy-code.md#what-happens-during-a-code-deploy).

### Request a worker type

> Your organization can enable up to 10 different worker types per cluster. Once a worker type is enabled for an Astro cluster, the worker type becomes available to any Deployment in that cluster and appears in the **Worker Type** menu of the Cloud UI.

Reviewer: ```suggestion
Your organization can enable up to 10 different worker types for each cluster. After a worker type is enabled on an Astro cluster, the worker type becomes available to any Deployment in that cluster and appears in the **Worker Type** menu of the Cloud UI.
```

1. Review the list of supported worker types for your cloud provider. See [AWS](resource-reference-aws.md#worker-node-types), [Azure](resource-reference-azure.md#worker-node-size-resource-reference), or [GCP](resource-reference-gcp.md#worker-node-size-resource-reference).
2. Contact [Astronomer support](https://cloud.astronomer.io/support) with the name of the worker type(s) you want to enable for your cluster. For example, `m6i.2xlarge`.

For more information on requesting cluster changes, see [Modify a cluster](modify-cluster.md).

## Create a worker queue

:::cli

If you prefer, you can also run the `astro deployment worker-queue create` command in the Astro CLI to create a worker queue. See the [CLI Command Reference](cli/astro-deployment-worker-queue-create.md).

:::

1. In the Cloud UI, select a Workspace and then select a Deployment.

> 2. Click the **Worker Queues** tab and then click **+ Worker Queue**.

Reviewer: ```suggestion
2. Click the **Worker Queues** tab and then click **Worker Queue**.
```

By default, all tasks run in the default worker queue. To run tasks on a different worker queue, assign the task to the worker queue in your DAG code.

To assign an Airflow task to a worker queue:

1. In the Cloud UI, select a Workspace and select a Deployment.

2. Click the **Worker Queues** tab.

3. Copy the name of the worker queue name you want to assign a task to.

> 4. In your DAG code, add a `queue='<worker-queue-name>'` argument to the definition of the task. If a task is assigned to a queue that does not exist or is not referenced properly, the task might get stuck in a `queued` state and fail to execute. Make sure that the name of the queue in your DAG code matches the exact name of the queue in the Cloud UI.

Reviewer: ```suggestion
4. In your DAG code, add a `queue='<worker-queue-name>'` argument to the definition of the task. If a task is assigned to a queue that does not exist or is not referenced properly, the task might remain in a `queued` state and fail to execute. Make sure that the name of the queue in your DAG code matches the name of the queue in the Cloud UI.
```

	For example, all instances of this task will run in the `short-running-tasks` queue:

	```python
	feature_engineering = DatabricksSubmitRunOperator(
		task_id='feature_engineering_notebook_task'
		notebook_task={
			'notebook_path': "/Users/{{ var.value.databricks_user }}/feature-eng_census-pred"
		},
		queue='short-running-tasks',
		)
	```

## Update a worker queue

:::cli

> If you prefer, you can also run the `astro deployment worker-queue update` command in the Astro CLI to update a worker queue. See the [CLI Command Reference](cli/astro-deployment-worker-queue-update.md).

Reviewer: ```suggestion
If you prefer, you can run the `astro deployment worker-queue update` command in the Astro CLI to update a worker queue. See the [CLI Command Reference](cli/astro-deployment-worker-queue-update.md).
```

:::

> You can update worker queue configurations at any time. This does not include worker queue name, which cannot be changed.

Reviewer: ```suggestion
You can update worker queue configurations at any time. The worker queue name can't be changed.
```

> If you need to change the worker type of an existing worker queue, Astronomer recommends doing so only at times which are not critical to production pipelines. Astronomer also recommends waiting up to 5 minutes after changing worker type before pushing new code to your Deployment.

Reviewer: ```suggestion
If you need to change the worker type of an existing worker queue, Astronomer recommends making the change at a time when it will not affect production pipelines. After you've changed a worker type, Astronomer recommends waiting a minimum of five minutes before pushing new code to your Deployment.
```
